Sources/TuistGenerator/Mappers/AutogeneratedProjectSchemeWorkspaceMapper.swift

```diff
@@ -5,11 +5,13 @@ import TuistGraph
 /// Mapper that generates a new scheme `ProjectName-Project` that includes all targets from a given workspace
 public final class AutogeneratedProjectSchemeWorkspaceMapper: WorkspaceMapping { // swiftlint:disable:this type_name
     let codeCoverageMode: CodeCoverageMode?
+    let autogenerationOptions: AutogenerationOptions?
 
     // MARK: - Init
 
-    public init(codeCoverageMode: CodeCoverageMode?) {
+    public init(codeCoverageMode: CodeCoverageMode?, autogenerationOptions: AutogenerationOptions?) {
```

Comment:
Shouldn't this just get a `TestingOptions`? I don't think it should be called when disabled

Reply:
It can be, there was a case where generation options is passed to the mapper even when set to disabled which confused things, but I can refactor to take testing options instead of the container

Reply:
I would refactor to avoid it 👍